common/lib/xmodule/xmodule/js/src/video/09_video_caption.js

```diff
@@ -210,7 +246,7 @@ function (Sjson, AsyncProcess) {
             var self = this,
                 state = this.state,
                 language = state.getCurrentLanguage(),
-                data, youtubeId;
+                data, youtubeId, results, start, captions;
```

Comment:
We can move `results`, `start`, and `captions` above and place each on their own line:

```javascript
var self = this,
    state = this.state,
    language = state.getCurrentLanguage(),
    results,
    start,
    captions,
    data, youtubeId;
```